test/methods/bools.jl

```diff
@@ -128,7 +128,7 @@ import GeometryOps as GO
         (-53.34136962890625, 28.430052892335723),
         (-53.57208251953125, 28.287451910503744),
     ]])
-	@test GO.overlaps(pl3, pl4) == false
+	@test GO.overlaps(pl3, pl4) == true # this was false before... why?
```

Comment:
This is the test that I changed!

Reply:
No idea why it was wrong, I copied these from Turf.jl. I guess testing these against LibGEOS is best?

Reply:
That makes sense! They were using intersects as equivalent to overlaps, which isn't valid I think since two of the same geometries on top of one another would have an intersection, namely the whole geometry. I will double check with LibGEOs though.